data/ApiDocs.md

```diff
@@ -119,102 +154,127 @@ the y coordinate. The pieces are: "wood", "ore", "gems", "bread", "gold", "piece
 this message.
 
 ### Conceding
 
 If a player concedes, they will send the message:
+
 ```json
 {"message":"Conceded", "color": string}
 ```
+
 Where color is the player's color. If the host concedes, they will randomly choose one of
 the remaining players, and will send the message:
+
 ```json
 {"message":"HostConcede", "oldColor": string, "newColor": string}
 ```
+
 newColor is the color of the new host player. The old host then prints the map state, if
 able, and an updated player turn order.
 
 If all players other than the host concede, then the host wins.
 
-### Handling Kicked Players
+## Handling Kicked Players
 
 If a player is kicked, then the host may choose one of the following options:
 
 #### Computer Plays
-Allow a computer to play for that player until they return. The host will send the 
-message:
-```json
-{"message":"KickedComputerStandin", "color": string, "newName": string}
-```
-color is the color of the kicked player. newName is a new username that the host may
-choose. It can be an empty string. 
 
-The next client that sends a JoinRequest for that color takes over that player. The 
-host will reply with:
-```json
-{"message":"JoinTakeover", "clientId": string}
-```
+Allow a computer to play for that player until they return. The host will send the 
+a `Replace` message.
 
 See *Taking Over for a Player* below for the next steps.
 
 #### Pause the game until they return.
 To choose this option, the host will send the message:
 
 ```json
-{"message":"KickedGamePause", "color": string}
+{"message":"GamePause", "pauseUTC": number}
 ```
-All game actions by the host will stop until the game resumes. Clients may still send 
-actions, and both clients and hosts must continue to ping each other.
+The `pauseUTC` field is the unix time in the UTC timezone when the server sends the message.
+It is used to ensure game pauses are idempotent.
 
-The host may then either choose to continue with either of the two other options. Or,
-when a new player joins successfully, the host will send the message:
+All game actions by the host will stop until the game resumes. If a client sends a message
+corresponding to a game action, the host will retransmit the `GamePause` message as
+its reply.
 
-```json
-{"message":"KickedGameResume", "color": string, "name": string}
-```
+The host may send any number of `Replace` messages while a game is paused.
 
-See *Taking Over for a Player* below for the next steps.
+When the host is ready to resume, they will send the message:
 
-#### Force the player to concede.
-The host can completely remove a player from the game by sending:
 ```json
-{"message":"KickedForcedConcede", "color": string}
+{"message":"GameResume", "pauseUTC": number}
 ```
-
-Play will then continue as normal. No pieces will be removed from the board.
+The `pauseUTC` field is the same as the corresponding `GameResume` message.

```

Comment:
Considering this is talking about the `GameResume` message, this might be a typo?
```suggestion
The `pauseUTC` field is the same as the corresponding `GamePause` message.

```

Reply:
Yup, I'll fix that.